### DISCUSSION TITLE
Where should I modify webpack values?

### DISCUSSION THREAD

**Discussion by Niicck** [react-static/react-static#1470] [General]
It looks like there are 2 places to modify webpack values. 

1. `static.config.js`
An example of this is for adding IE11 compatibility: https://github.com/react-static/react-static/blob/master/docs/concepts.md#browser-support
I might have the same question that was asked in this PR: https://github.com/react-static/react-static/pull/1461

2. `node.api.js`
I would like to add some additional plugins to the default webpack config. And the documentation says to do that within node.api.js: https://github.com/react-static/react-static/blob/master/docs/plugins/node-api.md#webpack It seems like this is where webpack values are being modified now: https://github.com/react-static/react-static/issues/1384. However, the browser-support documentation still says to modify static.config.js.

Which place should I put my custom webpack configs?
Do I modify config.entry within `static.config.js` but config.plugins within `node.api.js`? Could I modify config.entry and config.plugins within `node.api.js`? I tried to add plugins within `static.config.js`, but that did not seem to work.

**Comment by nddery** [comment]
Hi! I was not aware of the `webpack` property in `static.config.js`. I also can't find how this would work in the source code, and from #1461 it seems like using `node.api.js` would be the most future proof way to update the webpack config. We should probably update the documentation to use `node.api.js` over the `webpack` property in `static.config.js` so this is consistent.
